Automattic/vip-cli
# Dev Env: spawn WP-CLI as root to allow for FS operations

## Description

To allow for file operations-related commands we need to spawn the WP-CLI command under root, otherwise, we'll hit a variety of permissions-related errors.

Prior to the PR:

```
vip dev-env exec --slug 3453 -- wp plugin install akismet
Installing Akismet Spam Protection (4.1.12)
Downloading installation package from https://downloads.wordpress.org/plugin/akismet.4.1.12.zip...
The authenticity of akismet.4.1.12.zip could not be verified as no signature was found.
Unpacking the package...
Warning: Could not create directory.
```
With the PR:

```
 vip dev-env exec --slug 3453 -- wp plugin install akismet
Installing Akismet Spam Protection (4.1.12)
Downloading installation package from https://downloads.wordpress.org/plugin/akismet.4.1.12.zip...
The authenticity of akismet.4.1.12.zip could not be verified as no signature was found.
Unpacking the package...
Installing the plugin...
Plugin installed successfully.
Success: Installed 1 of 1 plugins.
```

## Steps to Test

Outline the steps to test and verify the PR here.

Example:

1. Check out PR.
1. Run `npm run build`
1. Create a new env
1. Verify ~cookies are delicious~ plugins are installable.
